File: Vigilantes.DaprWorkshop.VirtualBarista/csharp/Models/OrderSummary.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Vigilantes.DaprWorkshop.VirtualBarista.Models
{
    public class OrderSummary
    {
        [JsonProperty("orderId")]
        public Guid OrderId { get; set; }

        [JsonProperty("orderDate")]
        public DateTime OrderDate { get; set; }

        [JsonProperty("storeId")]
        public string StoreId { get; set; }

        [JsonProperty("customerName")]
        public string CustomerName { get; set; }

        [JsonProperty("loyaltyId")]
        public string LoyaltyId { get; set; }

        [JsonProperty("orderItems")]
        public List<OrderItemSummary> OrderItems { get; set; }

        [JsonProperty("orderTotal")]
        public decimal OrderTotal { get; set; }
    }
}

File: Vigilantes.DaprWorkshop.ReceiptGenerationService/java/receipt-service/src/main/java/vigilantes/daprworkshop/receiptservice/model/OrderSummary.java
package vigilantes.daprworkshop.receiptservice.model;

import java.math.BigDecimal;
import java.util.Date;
import java.util.List;
import java.util.UUID;

import lombok.Data;

@Data
public class OrderSummary {

    public UUID orderId;
    
    public Date orderDate;
    
    public String storeId;
    
    public String customerName;
    
    public String loyaltyId;
    
    public List<OrderItemSummary> orderItems;
    
    public BigDecimal orderTotal;
    
}